apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -49,55 +46,39 @@ interface UiItem {
   createUiItemsProvider: () => UiItemsProvider;
 }
 
-
-const SampleSpatialTree: React.FC = () => {
-  const iModel = useActiveIModelConnection();
-  return <>
-    {iModel && <SpatialContainmentTree
-      iModel={iModel}
-      groupByType={true}
-      setGroupByType={() => { return "group-by-type" }}
-      groupByDiscipline={true}
-      setGroupByDiscipline={() => { return "group-by-discipline" }}
-      displayGuids={true}
-      setIsDisplayGuids={() => { return "discard-guid-from-labe" }}
-      enableVisibility={true}
-      clipHeight={1.2}
-      clipAtSpaces={true}
-    />}
-  </>
-}
-const getSpatialTree = (): SelectableContentDefinition => {
-  return {
-    id: "spatial-containment-tree",
-    label: "containment",
-    render: () => (
-      <SampleSpatialTree />
-    ),
-  };
-};
-
 const configuredUiItems = new Map<string, UiItem>([
   [
     "tree-widget",
     {
-      initialize: async () => TreeWidget.initialize(),
+      initialize: async () => {
+        await BreakdownTrees.initialize();
+        await TreeWidget.initialize();
+      },
       createUiItemsProvider: () => new TreeWidgetUiItemsProvider({
-        additionalTrees: [getSpatialTree()]
+        additionalTrees: [{
+          id: "spatial-containment-tree",
+          label: "containment",
+          render: () => (
+            <SampleSpatialTree />
+          ),
+        }]
       }),
     }
   ],
   [
     "property-grid",
     {
-      initialize: async () => PropertyGridManager.initialize(),
+      initialize: async () => await PropertyGridManager.initialize(),
```

Comment:
you don't need to await on promise if you're returning it

Reply:
@Jake-Screen, this particular line is now fine, but I see you also removed `await` where you shouldn't have.

You don't need `await` if you're returning a promise:
- either
  ```ts
  () => myPromiseFunction()
  ```
- or
  ```ts
  () => { return myPromiseFunction(); }
  ```

You do need an await if you're not returning, e.g.:
```ts
() => {
  await myPromiseFunction();
}
```

Reply:
Looking at that initialize function, i see that it does return a promise, so doesn't it not need a await in front of it right there?

Reply:
changed